### DISCUSSION TITLE
Persist showing the emote menu button after typing

### DISCUSSION THREAD

**Discussion by AomariDev** [night/betterttv#6742] [Ideas]
![slika](https://github.com/night/betterttv/assets/5493998/aa3f51ea-d463-45bb-9c35-d59fd6ddac99)
![slika](https://github.com/night/betterttv/assets/5493998/a0939328-a482-482a-9261-f1fd56a971a2)

As seen in pictures above the emote menu button disappears after typing.

Inspecting and moving it from "picker-buttons" to "message-buttons" seems to make it stay even when typing.

![slika](https://github.com/night/betterttv/assets/5493998/1597358c-bc9b-42bb-a479-283ca6e56a61)

So could it be moved there or something?

**Comment by night** [comment]
This appears to be a bug, so we're closing this in reference to the issue #6746
